pangeo_forge_recipes/patterns.py

```diff
@@ -147,12 +148,20 @@ def __init__(
         fsspec_open_kwargs: Optional[Dict[str, Any]] = None,
         query_string_secrets: Optional[Dict[str, str]] = None,
         file_type: str = "netcdf4",
+        is_opendap: Optional[bool] = None,
     ):
         self.format_function = format_function
         self.combine_dims = combine_dims
         self.fsspec_open_kwargs = fsspec_open_kwargs if fsspec_open_kwargs else {}
         self.query_string_secrets = query_string_secrets if query_string_secrets else {}
         self.file_type = FileType(file_type)
+        if is_opendap:
+            _deprecation_message = (
+                "`FilePattern(..., is_opendap=True)` will be deprecated in v0.9.0. "
+                "Please use `FilePattern(..., file_type='opendap')` instead."
+            )
+            warnings.warn(_deprecation_message, DeprecationWarning)
+            self.file_type = FileType("opendap")
```

Comment:
👌

Reply:
In doing this I realized that we never actually followed through on 
https://github.com/pangeo-forge/pangeo-forge-recipes/blob/de33ebc1f71397be16bbe8110ed107fec2f300c7/pangeo_forge_recipes/recipes/xarray_zarr.py#L631-L634
https://github.com/pangeo-forge/pangeo-forge-recipes/blob/de33ebc1f71397be16bbe8110ed107fec2f300c7/pangeo_forge_recipes/recipes/xarray_zarr.py#L839-L840
but for clarity that's probably best as a separate PR.